.nvmrc

```diff
@@ -1 +1 @@
-16
+18
```

Comment:
So Cloudflare has started to support this version?

Reply:
OK, I've missed the commit description.

Reply:
Explained in f53764e621d41316d2b4b45de0be7959b2b990b8.